src/pages/EventDetailsPage.tsx

```diff
@@ -1,24 +1,110 @@
 import { ReactElement, useState } from "react";
 import { useParams } from "react-router-dom";
+import Frame from ".././assets/Frame.svg";
+import LinkSVG from ".././assets/Link.svg";
+import MeatballMenuSVG from ".././assets/MeatballMenu.svg";
+import PinSVG from ".././assets/Pin.svg";
+import TextIconSVG from ".././assets/TextIcon.svg";
 import { fetchEvent } from "../client/client";
 import { Event } from "../util/Types";
 
 export type EventDetailsPageProps = {
-  event?: Event;
+  e?: Event;
```

Comment:
I think `event` might be a better name since it has more meaning and is thus more readable